src/components/ReactSlider/ReactSlider.jsx

```diff
@@ -222,25 +222,26 @@ class ReactSlider extends React.Component {
 
         /**
          * Callback called before starting to move a thumb. The callback will only be called if the
-         * action will result in a change. The function will be called with one argument,
-         * the initial value(s).
+         * action will result in a change. The function will be called with two arguments, the first
+         * being the initial value(s) the second being thumb index.
          */
         // eslint-disable-next-line max-len
         // eslint-disable-next-line zillow/react/require-default-props, zillow/react/no-unused-prop-types
         onBeforeChange: PropTypes.func,
 
         /**
          * Callback called on every value change.
-         * The function will be called with one argument, the new value(s).
+         * The function will be called with two arguments, the first being the initial value(s) 
```

Comment:
Sorry to keep doing this to you, but now this line is also incorrect, should be "the first being the new value(s)".

Reply:
No problem, good catch ;)